index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Microverse Project- Survey form: Christian-Jesse</title>
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+  <div class="container">
+
+<header class="header">
+  <h1 id="title" class="text-center"> Survey Form</h1>
```

Comment:
There are too many IDs in the code. It would be better using classes instead of ID.

Reply:
- [ ] 😄 Of a truth, it is better to use classes selector over ID. Please kindly add the reason(s) for this required change to ensure that the other person(s) understand(s) you properly. 👍